### ISSUE TITLE
Multiple versions of @angular/common due to angularx-qrcode's dependencies

### ISSUE THREAD

**Issue by Gabweb** [Cordobo/angularx-qrcode#86]
One project I'm working on is using Angular 9.0.7. Since angularx-qrcode declares @angular/common as dependency, npm install@angular/common in two different versions. 

``` 
> npm ls @angular/common

+-- @angular/common@9.0.7
`-- angularx-qrcode@2.2.1
  `-- @angular/common@9.1.2
``` 

As stated by ng-packagr it's recommended to only use peerDependencies (https://github.com/ng-packagr/ng-packagr/blob/master/docs/dependencies.md). I therefore believe that @angular/common should be moved from the dependencies to the devDependencies of angularx-qrcode. With this change it would be installed for developing angularx-qrcode, but not in any project using angularx-qrcode.

In case you're accepting this change, I'm happy to contribute to this project and would create a PR.

**Comment by Cordobo** [comment]
Hi @Gabweb,

PRs are welcome, please open it against the develop branch.

**Comment by Cordobo** [comment]
Fixed in 2.3.4, thanks for contributing!
